docs/how-to/README.md

```diff
@@ -36,3 +36,4 @@
 - [How to use hotplug memory on arm64 in Kata Containers](how-to-hotplug-memory-arm64.md)
 - [How to setup swap devices in guest kernel](how-to-setup-swap-devices-in-guest-kernel.md)
 - [How to run rootless vmm](how-to-run-rootless-vmm.md)
+- [How to run Docker with Kata Containers](how-to-run-docker-with-kata.md)
```

Comment:
Nit: It might be clearer to call the file `how-to-run-docker-in-kata.md` (`with` -> `in`)?

Reply:
The link text `How to run Docker with Kata Containers` should keep the same as `How to run Docker in Docker with Kata Containers` in `how-to-run-docker-with-kata.md`.

And `run Docker with Kata Containers` may cause readers to confuse because now it's hard to `run Docker with Kata Containers` like the traditional way.

Reply:
Can we come to a consensus on this? I'm fine with the `with` -> `in` change but whether we should be referring to this as "Docker in Docker". 

Here are some options:

1. "How to run Docker in Kata Containers" (`how-to-run-docker-in-kata.md`)
  - Contextually correct, as we are running the Docker ecosystem in Kata
2. "How to run Docker in Docker with Kata Containers (`how-to-run-dind-with-kata.md`)
  - Familiar since most people refer to this process as "docker in docker" or "dind"
3. "How to run the Docker Daemon in Kata Containers" (`how-to-run-dockerd-in-kata.md`)
  - Most semantically correct -- we're running `dockerd`
4. "Nested containerization In Kata Containers" (`how-to-run-nested-containers-in-kata.md`)
  - High level view of what we're doing -- we're nesting containerization approaches (`containerd` & `dockerd`) with a VM (Kata Container) in the middle. Otherwise put, a Docker container in a Kata Container...